config/reader_test.go

```diff
@@ -369,104 +367,3 @@ func TestReadTLSFromCache(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 	})
 }
-
-func TestProcessConfigRegistersLogConfig(t *testing.T) {
```

Comment:
I will replace this test with a higher level one (something in the `web/server` package) once I get feedback on my other changes.